src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -410,18 +424,19 @@ IStreamsExecutor::Config IStreamsExecutor::Config::MakeDefaultMultiThreaded(cons
                        << streamExecutorConfig._threads_per_stream_small << ")";
     }
 #endif
-    const auto hwCores = !bLatencyCase && numaNodesNum == 1
-                             // throughput case on a single-NUMA node machine uses all available cores
-                             ? parallel_get_max_threads()
-                             // in the rest of cases:
-                             //    multi-node machine
-                             //    or
-                             //    latency case, single-node yet hybrid case that uses
-                             //      all core types
-                             //      or
-                             //      big-cores only, but the #cores is "enough" (pls see the logic above)
-                             // it is usually beneficial not to use the hyper-threading (which is default)
-                             : num_cores_default;
+    const auto hwCores =
+        !bLatencyCase && numaNodesNum == 1
+            // throughput case on a single-NUMA node machine uses all available cores
+            ? (!streamExecutorConfig._enable_hyper_thread ? num_cores_default : parallel_get_max_threads())
```

Comment:
Please use `(streamExecutorConfig._enable_hyper_thread ? parallel_get_max_threads() : num_cores_default)` to align with previous `(enable_hyper_thread ? parallel_get_max_threads() : getNumberOfCPUCores())`

Reply:
done